Userland/Libraries/LibArchive/Zip.cpp

```diff
@@ -121,8 +121,12 @@ static u16 minimum_version_needed(ZipCompressionMethod method)
 
 void ZipOutputStream::add_member(ZipMember const& member)
 {
+    char null_terminated_name[member.name.bytes_as_string_view().length() + 1];
+    memcpy(null_terminated_name, member.name.bytes_as_string_view().characters_without_null_termination(), member.name.bytes_as_string_view().length());
+    null_terminated_name[member.name.bytes_as_string_view().length()] = 0;
```

Comment:
Do we even need the null-termination? `LocalFileHeader` writes using the given length and data, there should be no `strlen` involved.